### PR TITLE
Update dockerfile and readme to include info for connecting to DB on localhost but outside of Docker

### PR DESCRIPTION
**Pull request by lewymd**
Updated dockerfile to include 

```
#    extra_hosts:
#    - "host.docker.internal:host-gateway"
```

commented out by default

Updated readme to show all information on what to do including getting docker IP if needed and query for creating user with the correct privileges

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -37,6 +37,11 @@ services:
       LOG_LEVEL: debug
       NO_BACKUP: 1
       NO_REQUIRE_ACCOUNT: 1
+      
+#Uncomment the 2 lines below if using as DB on your localhost outside of docker, then comment out or delete all of db and and depends_on:  -db lines
+#    extra_hosts:
+#    - "host.docker.internal:host-gateway"
```
Comment (clburlison) [inline docker-compose.yml]: You've interested this change into the middle of environment variables. You should add your three comments lines to after Line 60 in this PR.

Comment (lewymd) [reply docker-compose.yml]: good catch forgot about the commented variables, moved to below all of those.

**Comment by esauvisky** [comment]
Shouldn't we advise to comment out the entire `db:` service as well?

**Comment by lewymd** [comment]
> Shouldn't we advise to comment out the entire `db:` service as well?

Yup good point, only mentioned it in the docker-file not the readme. 

Added to readme.

**Pull request merged**